### DISCUSSION TITLE
docs/

### DISCUSSION THREAD

**Discussion by giscus** [ScratchAddons/website-v2#178] [Page comments]
# docs/

https://scratchaddons.com/docs/

**Comment by CST1229** [comment]
First e

**Reply by aaa464** [reply]
aaa

**Comment by tallypaws** [comment]
second e

**Comment by Ottiotta** [comment]
Please do not fill the comment thread with First, second ....

**Comment by asadullahexe** [comment]
hello

**Comment by asadullahexe** [comment]
how make clicker

**Comment by ressespufs** [comment]
can you add a unsave thing so that if you accidentally delete somthing and it auto-saves, you can go back.  P.S. thank you for reading this :)

**Reply by CST1229** [reply]
You can restore 1 sprite or costume with Edit -> Restore. by the way.

**Comment by kabss90** [comment]
Something it wrong with Scratch Addons: It does not work. Any addon I activate doesn't really get activated.

**Reply by kabss90** [reply]
I freaking meant something is wrong

**Reply by BroJac5246** [reply]
Can you provide a little more info about your environment? What browser, browser version, and operating system do you have, and are you on a school device or other device on which there are restrictions in place?

**Reply by BroJac5246** [reply]
Can you provide a little more info about your environment? What browser, browser version, and operating system do you have, and are you on a school device or other device on which there are restrictions in place?

**Comment by yushitoumingde-prog** [comment]
I am on iPad. I cannot do this.

**Comment by bytehedge** [comment]
i wish there was an addon that puts a setting on the fill bucket so it fills every instance of a selected color with the bucketed color. it would make palette swapping easier if you're trying to recolor a costume.

**Reply by CST1229** [reply]
hold shift while filling (at least in the bitmap editor). this works in vanilla scratch

**Comment by kimyoeuthen-byte** [comment]
There’s a bug when you go to the the add-ons menu is not Works

**Comment by mrhotdogrr** [comment]
You guys should add a frame picker so people who like doing animation can choose what mouth to use

**Reply by jessicakalynee123-hub** [reply]
mm;m
